chain/rosetta-rpc/src/lib.rs

```diff
@@ -700,14 +700,14 @@ fn get_cors(cors_allowed_origins: &[String]) -> CorsFactory {
 
 pub fn start_rosetta_rpc(
     config: crate::config::RosettaRpcConfig,
-    genesis: Arc<Genesis>,
+    genesis_config: GenesisConfig,
```

Comment:
Let's keep `Arc<Genesis>` here. Rosetta is optional, and disabled by default, and it is a trade-off we are willing to take for it. Rosetta exposes genesis records as a fake first block, so it needs those records.